feature_tests/src/attrs.rs

```diff
@@ -222,4 +222,26 @@ pub mod ffi {
             self.y /= o.y;
         }
     }
+
+    pub struct StructWithAttrs {
+        a: bool,
+        b: u32,
+    }
+
+    impl StructWithAttrs {
+        #[diplomat::attr(auto, constructor)]
+        pub fn new(a: bool, b: u32) -> Self {
+            Self { a, b }
+        }
+
+        #[diplomat::attr(supports = fallible_constructors, constructor)]
```

Comment:
This is still incorrect: the struct should only have one constructor. This gives the struct two.